src/sections/General/Navigation/utility/KanvasIcon.js

```diff
@@ -9,115 +9,35 @@ function KanvasIcon() {
       viewBox="0 0 218.18 350.43"
     >
       <g id="Layer_2" data-name="Layer 2">
-        <g id="Layer_1-2" data-name="Layer 1">
+        <g id="Layer_1-2" data-name="Layer 1" transform="scale(0.7) translate(-100, 30)">
```

Comment:
LGTM overall, thank you for changing this.
But I think if u need to reduce the size of svg, mb it will be better not to touch the transform for performance reasons, and play with viewBox property?